src/open_producten/producttypen/tests/api/test_prijs.py

```diff
@@ -49,17 +49,29 @@ def test_required_fields(self):
                 "actief_vanaf": [
                     ErrorDetail(string="Dit veld is vereist.", code="required")
                 ],
-                "prijsopties": [
-                    ErrorDetail(
-                        string="Er is minimaal één optie vereist.", code="invalid"
-                    )
-                ],
+                "prijsopties": [ErrorDetail("Dit veld is vereist.", code="required")],
```

Comment:
```suggestion
                "prijsopties": [ErrorDetail(_("This field is required."), code="required")],
```
This wil fail if the language is changed.